pkg/selinux/setroubleshoot-view.jsx

```diff
@@ -452,6 +455,15 @@ export class SETroubleshootPage extends React.Component {
             </cockpitListing.Listing>
         );
 
+        modifications = (
+            <Modifications
+                title={ _("System Modifications") }
+                emptyCaption={ this.props.selinuxStatus.permitted ? _("No System Modifications") : cockpit.format(_("The user <b>$0</b> is not permitted to view system modifications"), permission.user ? permission.user.name : '') }
```

Comment:
It seems like the permissions rendering and strings could move into the `Modifications` library. You should just pass a boolean whether it's permitted or not.

Reply:
The thinking why it is here was that the `Modifications` library should be neutral from any specific strings, so it could be later reused in other places (like firewall or so)